src/main/scala/org/reactivecouchbase/rs/scaladsl/Bucket.scala

```diff
@@ -56,6 +57,8 @@ class Bucket(config: BucketConfig, onStop: () => Unit) {
     (_bucket, _bucket.async(), _bucketManager, Future.successful(_bucket.async()))
   }
 
+  private val durationToExpiry = (expiration:Duration) => ((if (expiration < 30.days) 0L else System.currentTimeMillis / 1000L) + expiration.toSeconds).toInt
```

Comment:
The expiration is here added at the end of the expression. Only when it is greater that 30 days the unix timestamp of now must be added. Actually one can use always the absolute timestamp, but we still need the default expiration (which is actually 0) and means "no expiry" to be passed that way to the Java API, to be able to create document that do not expire. Rather that using 0 as "no expiry", it would be better and more convenient in Scala way, to use `Option[Duration]` as expiration in the API to make it clear, that it is possible to set "no expiry". What do you think?